en/docs/guides/login/oidc-parameters-in-auth-request.md

```diff
@@ -0,0 +1,149 @@
+#Use Advanced Parameters in Authentication Requests
+
+This page guides you through some special request parameters used with Open ID Connect authentication requests.
+
+ ----
+ 
+##[State parameter](../../../concepts/traditional-authentication-request)
+You can use this parameter when you need to maintain a correlation between the request and the response.
+
+```tab="Sample Request"
+https://<host>:/authorize?
+response_type=token
+&client_id=s6BhdRkqt3
+&state=xyz
+&redirect_uri=<callback_url>
+```
+
+```tab="Sample Response"
+HTTP/1.1 302 Found
+Location: <callback_url>#access_token=2YotnFZFEjr1zCsicMWpAA
+&state=xyz
+&token_type=bearer
+&expires_in=3600
+```
+ ----
+ 
+##[Nonce Parameter](../../authentication/concepts/traditional-authentication-request)
+You can use this parameter, when the client application needs to validate an ID Token issued by WSO2 Identity Server.
+The nonce claim embedded in the ID token must contain the exact same value that was sent in the request. 
+If not, authentication should be rejected by the application.
+
+```tab="Sample Request"
+https://<host>:9443/oauth2/authorize?response_type=id_token&client_id=NgTICXFPYnt7ETUm6Fc8NMU8K38a&redirect_uri=<callback_url>&nonce=abc&scope=openid
+```
+
+```tab="Sample Response"
+HTTP/1.1 302 Found
+Location: <callback_url>#access_token=2YotnFZFEjr1zCsicMWpAA
+&state=xyz
+&token_type=bearer
+&expires_in=3600
+```
+
+The decoded ID token is as follows.
+```{"auth_time":1453184484,"exp":1453188084,"sub":"admin@carbon.super","azp":"W2OoSxQDCVrBk1lnffo1NGCKZbQa","at_hash":"DoxjyXzmrL6Z_kWRzmBdCA","nonce":"abc","aud":["W2OoSxQDCVrBk1lnffo1NGCKZbQa"],"iss":"https:\/\/playground.local:9443\/oauth2\/token","iat":1453184484}```
+
+ ----
+ 
+##[Prompt Parameter](../../authentication/concepts/traditional-authentication-request)
+This parameter which can be sent with the authentication requests, can have the following three values.
+    
+    -   none         `
+    -   login         `
+    -   consent
+    
+###prompt=none
+The silent authentication can be initiated by using the prompt=none parameter with the authentication request.
+
+```tab="Sample Request"
+https://<host>:9443/oauth2/authorize?response_type=token&client_id=NgTICXFPYnt7ETUm6Fc8NMU8K38a&redirect_uri=<callback_url>&prompt=none&scope=openid
+```
+
+If the user has an already authenticated session and a pre-configured consent with the WSO2 Identity Server a successful response can be obtained as follows.
+
+Successful Response
+
+```tab="Sample Response"
```

Comment:
shall we add the tab title as "Successful Response" instead of "sample response" ?